distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -239,6 +244,16 @@
             <fileMode>0644</fileMode>
         </fileSet>
 
+        <!-- Custom library overrides -->
+        <fileSet>
+            <directory>${setup.folder}/libraryOverrides</directory>
+            <outputDirectory>
+                /system/org/apache/aries/blueprint/org.apache.aries.blueprint.core/1.8.2
+            </outputDirectory>
+            <includes>
+                <include>org.apache.aries.blueprint.core-1.8.2.jar</include>
```

Comment:
✏️ Instead of source controlling the jar can we just create our own minor version of this and upload to our nexus i.e org.apache.aries.blueprint.core-1.8.2_1.jar

Reply:
I'd tried with a `1.8.2_CODICE` version of the jar, but the dependency is buried within Karaf features files. Short of replacing all of those, there's no way to change the version it's requesting.

Reply:
❓ So how do we make sure that when I build locally, I'll get maven to download our forked version?

Reply:
When you build, you'd be building against the _real_ 1.8.2. The `ddf-common` assembly replaces it during zip construction.